File: code/patrick/python/lab03.py
total_amount = (input("\nEnter the total amount "))

converted_total_amount =  int(round(float(total_amount)*100))

print(converted_total_amount//25, "quarters")
converted_total_amont = converted_total_amount%25
print(converted_total_amount//100, "dimes")
converted_total_amount = converted_total_amount%10
print(converted_total_amount//5, "nickles")
converted_total_amount = converted_total_amount%5
print(converted_total_amount//1, "pennies")
converted_total_amount = converted_total_amount%1

File: code/patrick/javascript/lab03/lab03.js
let totalAmount = prompt('enter a number')

let convertedTotalAmount = parseFloat(totalAmount * 100)
console.log(convertedTotalAmount)

alert(Math.round(convertedTotalAmount/25), 'quarters')
convertedTotalAmount = convertedTotalAmount%25
alert(Math.round(convertedTotalAmount/10), 'quarters')
convertedTotalAmount = convertedTotalAmount%10
alert(Math.round(convertedTotalAmount/5), 'quarters')
convertedTotalAmount = convertedTotalAmount%5
alert(Math.round(convertedTotalAmount/1), 'quarters')
convertedTotalAmount = convertedTotalAmount%1
